csep/utils/csep1_unit_tests

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jan 27 14:38:30 2020
+Implementation of Unit Test for L Test
+@author: khawaja
+"""
+import numpy
+import unittest
+from csep1_tests import n_test
+from csep1_tests import poisson_log_likelihood
+from csep1_tests import _forecast_realization_inverse_cdf
+from csep1_tests import _m_test_prep
+from csep1_tests import _s_test_prep
+from csep1_tests import _conditional_l_test_prep
+
+
+class TestGriddedForecastTests(unittest.TestCase):
```

Comment:
separate class for each test type or you can break up the unit tests so they only test a single aspect of the function.